CONTRIBUTING.md

```diff
@@ -4,135 +4,135 @@
 
 ## Which Languages can I Contribute?
 
-  - *Any language* is welcomed. *Anyone* is welcomed to contribute. :earth_asia: :earth_americas: :earth_africa:
-  - There are several [issues](//github.com/liuderchi/atom-i18n/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) require your help.
-  - NOTE: Currently the author do not review grammar of the translation.
-
+- _Any language_ is welcomed. _Anyone_ is welcomed to contribute. :earth_asia: :earth_americas: :earth_africa:
+- There are several [issues](//github.com/liuderchi/atom-i18n/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) require your help.
+- NOTE: Currently the author do not review grammar of the translation.
```

Comment:
There is a grammar error, it should say:

"NOTE: Currently the author do**es** not review grammar of the translation."

Reply:
Thanks! fixed in 9b87a88